src/pages/HomePage/HomePage.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FC } from "react";
+import styled from "@emotion/styled";
+
+import { useAppConfigContext } from "../..";
+
+import { HomeSearchBar } from "../../components/Layout/HomeSearchBar";
+
+const Container = styled.div({
+  // Center the children horizontally and vertically
+  display: "flex",
+  justifyContent: "center",
+  alignItems: "center",
+});
+
+const HomePage: FC = () => {
+  const { municipality } = useAppConfigContext();
+
+  return (
+    <Container>
+      <h1>{`Discover ${municipality.name} city council meetings, legislations, and members`}</h1>
+      <HomeSearchBar />
```

Comment:
Any translation suggestions for this string?

Reply:
Looks good to me.  Maybe add "debates" since those can be added too?

Reply:
I meant what would the Spanish and German translation of this string be?

Reply:
I think adding 'debates' will make this string too long. Would "Discover Seattle, WA city council` be good enough? The different type of searching(meetings, legislations, and members strings) are available in the advanced options.

Reply:
Eh just going to change it "Search City Council" and modify the "Search City Council Transcripts" value in `strings`. The municipality name is already mentioned in the app's header. We can improve the home page later.